src/providers/boleto-api-caixa/index.js

```diff
@@ -118,6 +119,13 @@ const buildPayload = (boleto, operationId) => {
         stateCode: formattedRecipientStateCode,
       },
     },
+    payeeGuarantor: {
+      name: path(['company_name'], boleto),
+      document: {
+        type: getDocumentType(path(['company_document_number'], boleto)),
+        number: path(['company_document_number'], boleto),
```

Comment:
opa, visto que o campo company_document_number não é obrigatório na super boleto, vale usarmos o pathOr

```suggestion
        number: pathOr('',['company_document_number'], boleto),
```

Reply:
como estamos pegando o nó `company_document_number `duas vezes de boleto, podemos pegar uma vez só, igual o `companyName `da linha 88, e usar para os atributos do `payeeGuarantor` 😀

Reply:
feito